rest-service/manager_rest/blueprints_manager.py

```diff
@@ -155,13 +155,14 @@ def create_snapshot(self, snapshot_id,
         )
         return execution
 
-    def restore_snapshot(self, snapshot_id):
+    def restore_snapshot(self, snapshot_id, restore_deployment_workers):
```

Comment:
I would rename it everywhere to <b>recreate_dep_envs</b> or someting similar, because it is not only about workers (adding depl config to Riemann as well for example) - it is simply creating deployments' environments so let the name suggest it.